backend-lib/src/main/rust/lib.rs

```diff
@@ -2036,12 +2025,12 @@ pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_createPro
         let network = parse_network(network_id as u32)?;
```

Comment:
On line 2024 the wrong method name is used in the `info_span!`.

Reply:
I checked all other uses of `info_span!` and made comments for any inconsistencies.